### ISSUE TITLE
Using WEST database specific library

### ISSUE THREAD

**Issue by jhillairet** [piScope/piScope#33]
Few year ago I've written a bad hack of the job runner in order to use the specific TS/WEST database library instead of MDSplus. it worked, but at the price of loosing the compatibility with many thinks related to MDSplus. 

Since I've still have problems related with MDSplus access, which seems due from our MDSplus <-> TS/WEST db MDSplus wrapper, I would like to retry using TS/WEST db library. How could I do ?

Basically, somewhere I should return a couple of _values_ and _time_ arrays from our internal db library instead of using MDSplus command _gettsbase()_. In direct_jobrunner_ts.py ?
